### PR TITLE
refactor: Restructure project for single import

### PR DESCRIPTION
**Pull request by Peetee06**
This PR restructures the project to align with [official  Flutter recommendations](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#structure). 

Using the package now requires only a single import:
```dart
import 'package:stomp_dart_client/stomp_dart_client.dart';
```
This makes working with `stomp_dart_client.dart` a little bit easier, because one can use things like VSCode code actions to automatically add an import to the package.

One note:
I couldn't run the tests in `stomp_test.dart` locally, because the `spawnHybridCode` method doesn't use the project's `.dart_tool` folder but rather a global one. The global one does not have the necessary packages installed, which is why the hybrid code can't be run. I couldn't find a way to fix this other than installing the necessary packages globally, which I don't want to do.

Please let me know if any further adjustments are desired. I'll be happy to fine-tune this PR further.

### CODE REVIEW

**Comment by KammererTob** [comment]
Thanks for the PR, i will have a look in the next few days.

**Comment by KammererTob** [comment]
@Peetee06 I tried to run the tests locally, but i ran into some issues. Maybe you have guidance for this:
In `stomp_config_test.dart` i do this: `await platform.connect(config..resetSession());`.

This currently throws the following error:
```
test/stomp_config_test.dart:23:34:
Error: The argument type 'StompConfig/*1*/' can't be assigned to the parameter type 'StompConfig/*2*/'.
 - 'StompConfig/*1*/' is from 'package:stomp_dart_client/src/stomp_config.dart' ('lib/src/stomp_config.dart').
 - 'StompConfig/*2*/' is from 'lib/src/stomp_config.dart'.
          await platform.connect(config..resetSession());
                                 ^
Error: Compilation failed.
```

If i try to use relative path for imports VSC complains (probably rightly so), that the path is not supposed to be relative.

> Can't use a relative path to import a library in 'lib'.
> Try fixing the relative path or changing the import to a 'package:' import.dart [avoid_relative_lib_imports](https://dart.dev/lints/avoid_relative_lib_imports)

Do you happen to know what to do with unit-tests?
Thanks!

**Comment by Peetee06** [comment]
@KammererTob, thanks for reporting this. It seems like dart couldn't see, that `_connect_io.dart` imports the same StompConfig. I fixed the import and the test succeeds locally now. Let me know if they succeed for you as well.

File: test/stomp_test.dart [hunk 1]
```diff
@@ -23,7 +19,7 @@ void main() {
         import 'package:web_socket_channel/io.dart';
         import 'package:stomp_dart_client/stomp_parser.dart';
```
Comment (KammererTob) [inline test/stomp_test.dart:20]: Can you also please update those imports? Those are still throwing an error. There are a few in the hybrid tests.

**Review by KammererTob — CHANGES_REQUESTED** [review]
I have added a comment to one file, but this affects all files with hybrid tests. Can you please adjust those? For me they threw an error while testing.

Overall this is a breaking change, right? So i guess this would land in 2.0 then

File: test/stomp_test.dart (hunk 1 above)
Comment (Peetee06) [reply test/stomp_test.dart]: good catch! Should be fixed, let me know if I missed any place. As I said in the PR's description I can't easily run these tests myself..

**Comment by Peetee06** [comment]
@KammererTob true, this is a breaking change. Thank you for reviewing!

**Review by KammererTob — APPROVED** [review]

**Comment by KammererTob** [comment]
Thank you. Looks good now. I will wait a few weeks if anything else comes up for a new release. I will release this as 2.0.0 then.

**Comment by Peetee06** [comment]
Very nice! Thank you for creating this project! Happy that I got to contribute!

**Pull request merged**